resources/public/api/conf/6.0/loss_claims_delete.yaml

```diff
@@ -1,83 +1,84 @@
-summary: Delete a Loss Claim
-description: |
-  This endpoint allows a developer to delete an existing Loss Claim.
-  A National Insurance number and Claim ID must be provided.
+delete:
+  summary: Delete a Loss Claim
+  description: |
+    This endpoint allows a developer to delete an existing Loss Claim.
+    A National Insurance number and Claim ID must be provided.
+    
+    ### Test data
+    Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+    
+    | Header Value (Gov-Test-Scenario) | Scenario                                       |
+    | -------------------------------- | ---------------------------------------------- |
+    | N/A - DEFAULT                    | Simulate success scenario                      |
+    | NOT_FOUND                        | Simulates the scenario where no data is found. |
+    | OUTSIDE_AMENDMENT_WINDOW         | Simulates the scenario where request cannot be completed as it is outside the amendment window. |
+    | STATEFUL                         | Performs a stateful delete.                    |
   
-  ### Test data
-  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
-  
-  | Header Value (Gov-Test-Scenario) | Scenario                                       |
-  | -------------------------------- | ---------------------------------------------- |
-  | N/A - DEFAULT                    | Simulate success scenario                      |
-  | NOT_FOUND                        | Simulates the scenario where no data is found. |
-  | OUTSIDE_AMENDMENT_WINDOW         | Simulates the scenario where request cannot be completed as it is outside the amendment window. |
-  | STATEFUL                         | Performs a stateful delete.                    |
-
 
-tags:
-  - Loss Claims
+  tags:
+    - Loss Claims
 
-parameters:
-  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
-  - $ref: './common/pathParameters.yaml#/components/parameters/claimId'
-  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
-  - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
-  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+  parameters:
+    - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+    - $ref: './common/pathParameters.yaml#/components/parameters/claimId'
```

Comment:
Please add the tax year as part of the path parameters below i.e. include [- $ref: './common/pathParameters.yaml#/components/parameters/taxYearClaimedFor'](https://github.com/search?q=repo%3Ahmrc%2Findividual-losses-api+-+%24ref%3A+%27.%2Fcommon%2FpathParameters.yaml%23%2Fcomponents%2Fparameters%2FtaxYearClaimedFor%27&type=code), thanks